package.json

```diff
@@ -32,11 +37,17 @@
     "limiter": "^1.1.3",
     "lodash": "^4.17.15",
     "mariadb": "^2.0.1-beta",
-    "memory-cache": "^0.2.0"
+    "memory-cache": "^0.2.0",
+    "dotenv": "^8.2.0"
```

Comment:
what does this give us?

Reply:
It gives us a consistent way of setting environment variables. No matter what is your local OS or we are moving our server from one to another.